mws/resource_mws_private_access_settings_test.go

```diff
@@ -51,6 +51,51 @@ func TestResourcePASCreate(t *testing.T) {
 	assert.Equal(t, "abc/pas_id", d.Id())
 }
 
+func TestResourcePASCreate_PublicAccessDisabled(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "POST",
+				Resource: "/api/2.0/accounts/abc/private-access-settings",
+				ExpectedRequest: PrivateAccessSettings{
+					AccountID:           "abc",
+					Region:              "ar",
+					PublicAccessEnabled: false,
+					PasName:             "pas_name",
+					PrivateAccessLevel:  "ACCOUNT",
+					ForceSendFields:     []string{"PublicAccessEnabled"},
+				},
+				Response: PrivateAccessSettings{
+					PasID: "pas_id",
+				},
+			},
+			{
+				Method:   "GET",
+				Resource: "/api/2.0/accounts/abc/private-access-settings/pas_id",
+
+				Response: PrivateAccessSettings{
+					AccountID:           "abc",
+					PasID:               "pas_id",
+					PublicAccessEnabled: false,
+					Region:              "ar",
+					PasName:             "pas_name",
+					ForceSendFields:     []string{"PublicAccessEnabled"},
```

Comment:
is it really returned as answer?

Reply:
Do you mean the ForceSendFields?
It is not returned per se. The mocking library does not return the object itself (it's here just for convenience), but instead will marshall this object and "return" the json, which then is deserialised.
We need to add it here so that the returned json contains the relevant field.